master/getting-started/mesos/tutorials/launching-tasks.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Launching Tasks
+---
+
+The following information describes how to launch Calico networked tasks in Mesos
+using Marathon.
+
+### Unified Containerizer
+
+Launch a Unified Containerizer task onto a Calico network by setting
```

Comment:
I don't agree with these suggestions. They'll just bloat the page up with more stuff to read.

I'd rather this just be a reference of actions, than a tutorial that prefaces everything with "In the following example".